doc/users/next_whats_new/polyquadmesh.rst

```diff
@@ -0,0 +1,23 @@
+``PolyQuadMesh`` is a new class for drawing quadrilateral meshes
+----------------------------------------------------------------
+
+``plt.pcolor()`` now returns a ``PolyQuadMesh`` class. It is a
```

Comment:
link to the class?

Reply:
I think it's helpful to point out what _used_ to be returned and maybe any major usage changes we expect the user to see (if any)

Reply:
I updated it with references to what used to be returned `PolyCollection`, and tried to indicate that we are subclassing `PolyCollection` still, so things should hopefully be transparent to the end-user.